ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -421,3 +421,49 @@ class TestBackupReinstallRestoreWithKRA(BaseBackupAndRestoreWithKRA):
     def test_full_backup_reinstall_restore_with_vault(self):
         """backup, uninstall, reinstall, restore"""
         self._full_backup_restore_with_vault(reinstall=True)
+
+
+class TestBackupNotFailesSilently(IntegrationTest):
+    """Before the fix, when ipa-backup was called for the first time,
```

Comment:
Traditionally, a doc string starts with a short title that fits into one line:

```
"""Short title

Long text
"""
```

Reply:
This still wasn't fixed.